app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
```

Comment:
Done. And I took the opportunity to rebase the `markdown-support` branch on top of `master`.